owl/EASE-SAY.owl

```diff
@@ -123,6 +277,83 @@
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE-SAY.owl#ExecutableSchematicTheory">
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/EASE-SAY.owl#SchematicTheory"/>
+        <rdfs:comment>Also known as &quot;executing schemas&quot; or &quot;x-schemas&quot;, these were defined by Bergen and Chang in their work &quot;Embodied Construction Grammar in Simulation-Based Language Understanding&quot; as:
+
+&quot;Executing schemas, or x-schemas, are dynamic representations motivated in part by motor and perceptual systems (Bailey 1997; Narayanan 1997), on the assumption that the same underlying representations used for executing and perceiving an action are brought to bear in understanding language about that action. The
+x-schema formalism is an extension of Petri nets (Murata 1989) that can model sequential, concurrent, and
+asynchronous events&quot;
+
+In SOMA, we do not restrict the formalism of ExecutableSchematicTheories; i.e. they don&apos;t have to be Petri Nets.
+
+They maintain their role however as representations able to drive a simulation, at some level of abstraction, of an embodied action. This level of abstraction may be still fairly underspecified as in the case of the original x-schemas and as such not a plan that an agent could run in an actual physical environment without further information.</rdfs:comment>
+    </owl:Class>
+    
+
+
+    <!-- http://www.ease-crc.org/ont/EASE-SAY.owl#FunctionalSpatialSchemaTheory -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE-SAY.owl#FunctionalSpatialSchemaTheory">
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/EASE-SAY.owl#ImageSchemaTheory"/>
+        <rdfs:subClassOf>
+            <owl:Restriction>
+                <owl:onProperty rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#defines"/>
+                <owl:someValuesFrom rdf:resource="http://www.ease-crc.org/ont/EASE-SAY.owl#LocatumRole"/>
+            </owl:Restriction>
+        </rdfs:subClassOf>
+        <rdfs:subClassOf>
+            <owl:Restriction>
+                <owl:onProperty rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#defines"/>
+                <owl:someValuesFrom rdf:resource="http://www.ease-crc.org/ont/EASE-SAY.owl#RelatumRole"/>
+            </owl:Restriction>
+        </rdfs:subClassOf>
+        <rdfs:comment rdf:datatype="http://www.w3.org/2001/XMLSchema#string">GUM-4-space: The most general type of functional spatial relations.</rdfs:comment>
+    </owl:Class>
+    
+
+
+    <!-- http://www.ease-crc.org/ont/EASE-SAY.owl#GoalRole -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE-SAY.owl#GoalRole">
+        <rdfs:subClassOf rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#Role"/>
+        <rdfs:comment rdf:datatype="http://www.w3.org/2001/XMLSchema#string">Denotes the destination of a route. Typically represented by a FunctionalSpatialSchema.</rdfs:comment>
```

Comment:
Why not name it "Destination" then?
"Goal" seems to be more broad.

Reply:
The naming of "Goal" is simply due to cognitive linguistics heritage, but we can of course also merge it to the existing concept.

Reply:
But for me it seems a destination is just one example of a goal. Another one would be to win a soccer tournament, or to achieve a degree, etc which would rather count as "destination" only metaphorically.

Reply:
FYI: the term "Goal" is used in DUL for plans, as in "plans are descriptions having an explicit goal". In that definition a "Goal" is a description of a (desired) situation.

Reply:
Ah, thanks for clarifying! In light of this definition, it makes sense to rename and merge `GoalRole` with the more primitive `Destination` concept.